### ISSUE TITLE
`heal_cols': undefined method `begin' for nil:NilClass (NoMethodError)

### ISSUE THREAD

**Issue by aditiamahdar** [adworse/iguvium#1]
Hi,

Thank you for creating very useful gem,
I've tried to fetch table from inside a PDF, it works very nice.

But I found issue when fetching some PDF, it result this error:
```
7: from /Users/mac/.rvm/gems/ruby-2.5.3/gems/iguvium-0.9.0/lib/iguvium/page.rb:49:in `extract_tables!'
	 6: from /Users/mac/.rvm/gems/ruby-2.5.3/gems/iguvium-0.9.0/lib/iguvium/page.rb:78:in `recognize!'
	 5: from /Users/mac/.rvm/gems/ruby-2.5.3/gems/iguvium-0.9.0/lib/iguvium/page.rb:78:in `map'
	 4: from /Users/mac/.rvm/gems/ruby-2.5.3/gems/iguvium-0.9.0/lib/iguvium/page.rb:78:in `block in recognize!'
	 3: from /Users/mac/.rvm/gems/ruby-2.5.3/gems/iguvium-0.9.0/lib/iguvium/page.rb:78:in `new'
	 2: from /Users/mac/.rvm/gems/ruby-2.5.3/gems/iguvium-0.9.0/lib/iguvium/table.rb:20:in `initialize'
	 1: from /Users/mac/.rvm/gems/ruby-2.5.3/gems/iguvium-0.9.0/lib/iguvium/table.rb:73:in `heal'
/Users/mac/.rvm/gems/ruby-2.5.3/gems/iguvium-0.9.0/lib/iguvium/table.rb:85:in `heal_cols': undefined method `begin' for nil:NilClass (NoMethodError)
```

Could anyone help me with this issue?

**Comment by adworse** [comment]
Looks like Iguvium can't detect any cells and fails. It shouldn't be like this anyway, I will fix this as soon as you will provide me with PDF's causing this error.

**Comment by aditiamahdar** [comment]
Sorry I can't provide the pdf, since it's confidential doc, I ended up with add `rescue` to `heal_cols` method. Somehow the doc header is a table and it cause this error, but the real table content is work fine.

**Comment by adworse** [comment]
I’ll try to figure it out and make a fix. Would you like to test a local version of gem on your files before I roll the fix in production?

**Comment by adworse** [comment]
Actually, there is a way to show me the root of the problem without disclosing the document. Could you please run `Iguvium::CV.new(Iguvium::Image.read(path_to_file, page_number))` on the problematic page and show me the resulting Struct? It only contains recognized borders, so no sensitive info will be shown.

**Comment by aditiamahdar** [comment]
I've tried to run that command, and resulting this error:
```
Errno::ENOENT: No such file or directory @ rb_sysopen - FILENAME.rgb
from /Users/mac/.rvm/gems/ruby-2.5.3/gems/chunky_png-1.3.11/lib/chunky_png/datastream.rb:70:in `initialize'
```

**Comment by nangirril** [comment]
I have the same problem. Could this be due to the fact that the PDF file contains all as recognized text?

**Comment by adworse** [comment]
> ```
> Errno::ENOENT: No such file or directory @ rb_sysopen - FILENAME.rgb
> ```

This is probably due to non-ASCII symbols in the filename or in the path, or `irb` running from the different directory. Could you try to place the file in some very straightforward path, no spaces etc., and run irb from the same folder?

**Comment by adworse** [comment]
Fixed in 0.9.1
